HACKING.md

```diff
@@ -5,6 +5,12 @@
 First install a few dependencies:
 
     sudo apt install gcc g++ make python3-dev python3-venv libffi-dev libsodium-dev libapt-pkg-dev libarchive13 squashfs-tools patchelf
+    
+   
+Fork the repository snapcraft on https://github.com/snapcore/snapcraft
```

Comment:
I'm not sure I agree with @elopio, here. This is in the "installing from pip" section. The "installing for development" section is below. Telling people to fork here seems premature. I'd rather see it below.

Reply:
As long as it's before the call that uses requirements.txt, it works for me.